packages/one-app-hmr/lib/locale.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import path from 'path';
+import chokidar from 'chokidar';
+import compileModuleLocales from '../../one-app-locale-bundler/src/compileModuleLocales';
+
+export async function buildHotModuleLocale(modulePath) {
+  await compileModuleLocales(modulePath);
```

Comment:
Not the best way for us to rebuild locales, since any single locale updated will rebuild the entirety of locales for that module. We should come up with a better solution. Also it's log output is a bit "extra" for HMR/watch, but great for single builds.